src/extension-test/unit/cursor-doc/cursor-context-test.ts

```diff
@@ -0,0 +1,224 @@
+import * as expect from 'expect';
+import * as context from '../../../cursor-doc/cursor-context';
+import { docFromTextNotation, textAndSelection } from '../common/text-notation';
+
+describe('Cursor Contexts', () => {
```

Comment:
@bpringe @crispinb This is what the tests can look like with the helper function I made last week. Pointing at it because I am so pleased with the result! 😄

Reply:
I love it!